org.sonarlint.eclipse.sloop.no-arch/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,11 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %bundle_name
+Bundle-SymbolicName: org.sonarlint.eclipse.sloop.no-arch;singleton:=true
+Bundle-Version: 9.4.0.qualifier
+Eclipse-PlatformFilter: (& (osgi.os=win32) (osgi.arch=aarch64))
```

Comment:
is this valid for no arch?

Reply:
Yes. There is currently no option to have a filter that says "everything else." I opened a [discussion](https://github.com/eclipse-pde/eclipse.pde/discussions/1026) at Eclipse PDE some time ago.

Currently, we only allow macOS, Linux, and Windows with the architectures x86_64 and aarch64 (ARM). The only one we have not with a Java distribution is Windows on ARM, because there basically is none.